specs/protocol/superchain-configuration.md

```diff
@@ -64,3 +67,55 @@ When the Pause is activated, the following methods are disabled:
 1. `StandardBridge.finalizeBridgeERC20()`
 1. `StandardBridge.finalizeBridgeETH()`
 1. `L1ERC721Bridge.finalizeBridgeERC721()`
+
+## Dependency manager
+
+The `SuperchainConfig` contract will manage and keep track of the dependency graph.
+It will be queried as the source of truth to get which chains are part of the Superchain.
+It will also allow to add a chain to the op-governed cluster and update each chain’s dependency set.
+
+### Interface and properties
+
+The contract will add the following storage layout and function:
+
+**`SHARED_LOCKBOX`**
```

Comment:
Can you define the functions more similar to this? https://github.com/ethereum-optimism/specs/blob/main/specs/experimental/custom-gas-token.md#depositerc20transaction